samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -348,24 +353,24 @@ static void fmdn_provisioning_state_changed(bool provisioned)
 
 	/* Triggered on the unprovisioning operation. */
 	if (factory_reset_executed) {
-		LOG_INF("Please press a button to put the device in the Fast Pair discoverable "
-			"advertising mode after a reset to factory settings");
+		LOG_INF("The device was unprovisioned and reset to factory setting");
+
+		/* Disable advertising requests related to the FMDN. */
+		app_fp_adv_request(fp_adv_trigger_ak_write, false);
+		app_fp_adv_request(fp_adv_trigger_clock_sync, false);
+
+		/* Disable advertising request from the UI. */
+		fp_adv_ui_request = false;
+		app_fp_adv_request(fp_adv_trigger_ui, fp_adv_ui_request);
```

Comment:
At this point, the fp_adv module is already active and advertising. We do not want to trigger a pop-up right after notification, we need to disable advertising when the factory reset is scheduled: `if (fp_account_key_present != provisioned) {`